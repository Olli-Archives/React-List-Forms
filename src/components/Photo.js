import React from 'react';
import PropTypes from 'prop-types';

export default function Photo({ photo }){
  const photoToRender = photo;
  return (<img src={photoToRender}/>);
}


Photo.propTypes = {
  photo: PropTypes.string.isRequired
}
;
