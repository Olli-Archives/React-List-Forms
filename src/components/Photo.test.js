import React from 'react';
import Photo from './Photo';
import { shallow } from 'enzyme';

const photoUrl = 'test.url.com';

describe('Photo component', ()=>{
  it('renders photo', ()=>{
    const wrapper = shallow(<Photo photo={photoUrl}/>);
    expect(wrapper).toMatchSnapshot();
    
  });
})
;
