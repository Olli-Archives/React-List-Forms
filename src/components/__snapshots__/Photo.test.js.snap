// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Photo component renders photo 1`] = `
<img
  src="test.url.com"
/>
`;
